copying.md

```diff
@@ -109,6 +109,7 @@ _the openage authors_ are:
 | Arne Sellmann               | PythonicChemist             | arne dawt sellmann à gmx dawt de                  |
 | Rafael X. Morales Georgi    | chocoladisco                | chocoladisco à gmail dawt com                     |
 | Marcel Schneider            | schnema123                  | marcelschneider5 à outlook dawt de                |
+| Samuel Grigolato            | samuelgrigolato             | samuel dawt grigolato à gmail dawt com                |
```

Comment:
The rightmost border of the table has to line up

Reply:
Fixed!